core/modules/commands/makelibrary.js

```diff
@@ -54,6 +54,22 @@ Command.prototype.execute = function() {
 	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.pluginsPath));
 	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.themesPath));
 	collectPlugins(path.resolve($tw.boot.corePath,$tw.config.languagesPath));
+	// Adding local wiki paths
+	try {
+		collectPlugins(path.resolve($tw.boot.wikiPath,$tw.config.wikiPluginsSubDir));
+	} catch (ex) {
+		console.log("No Local Plugins Folder");
```

Comment:
Hi,

The catch is for the nominal case where there not be plugins, themes and/or languages folders in the current wiki. Can you suggest a more elegant way to test the existence of the folders? Perhaps we can just eliminate the log messages to be coherent with the relative silence of the tiddlywiki command. I considered through the directories with a fs.lstat() call. I am not fond of ifs. 

Take care,

Christopher Mann

> On 12 Nov 2015, at 08:21, Tobias Beer <notifications@github.com> wrote:
> 
> In core/modules/commands/makelibrary.js:
> 
> > @@ -54,6 +54,22 @@ Command.prototype.execute = function() {
> >  	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.pluginsPath));
> >  	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.themesPath));
> >  	collectPlugins(path.resolve($tw.boot.corePath,$tw.config.languagesPath));
> > +	// Adding local wiki paths
> > +	try {
> > +		collectPlugins(path.resolve($tw.boot.wikiPath,$tw.config.wikiPluginsSubDir));
> > +	} catch (ex) {
> > +		console.log("No Local Plugins Folder");
> I see how you may used that for testing, but do we need any such error catching in the final code?
> 
> —
> Reply to this email directly or view it on GitHub.
>